src/allmydata/storage/server.py

```diff
@@ -823,7 +824,7 @@ def __init__(self, storage_server):  # type: (StorageServer) -> None
         self._server = storage_server
 
         # Canaries and disconnect markers for BucketWriters created via Foolscap:
-        self._bucket_writer_disconnect_markers = {}  # type: Dict[BucketWriter,Tuple[IRemoteReference, object]]
+        self._bucket_writer_disconnect_markers : dict[BucketWriter, tuple[IRemoteReference, Any]] = {}
```

Comment:
I'm not sure I know why `object` is being replaced with `Any` here.

Everything is an `object` and you can't do anything with it.  That seems suitable for this marker object that comes from Foolscap and we should only ever give back to Foolscap.

Everything is an `Any` and you can do anything with it.  But it should probably be an error if we try to do anything with it.